org.eclipse.xtext.ui.codetemplates/src-gen/org/eclipse/xtext/ui/codetemplates/services/CodetemplatesGrammarAccess.java

```diff
@@ -94,8 +95,8 @@
 		//	keywordContext=STRING) body=TemplateBodyWithQuotes;
 		@Override public ParserRule getRule() { return rule; }
 		
-		//name=ValidID '(' id=ID ',' description=STRING ')' 'for' (context=[xtext::AbstractRule|ValidID] | keywordContext=STRING)
-		//body=TemplateBodyWithQuotes
+		//name=ValidID '(' id=ID ',' description=STRING ')' 'for' (context=[xtext::AbstractRule|ValidID] | keywordContext=STRING
+		//) body=TemplateBodyWithQuotes
```

Comment:
Weird formatting @ArneDeutsch

Reply:
Ja, in der Tat. Habe es im Unit-Test versucht nachzustellen, da kommt aber raus was man erwarten würde ...

Reply:
Ich kann mir nur vorstellen, dass da gerade so die maximale Zeichenanzahl erreicht ist und der automatisch umbricht. Wenn ich die Zeichen zähle komme ich auf 117 ... bissl Whitespace davor ...